frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -19,8 +60,181 @@ function AuthedView() {
     <dl>
       <h1>
         <MetaTitle title="Dashboard" />
-        Welcome, <span style={{ color: "dodgerblue" }}>{user.username}</span>
+        <ProjectList />
       </h1>
     </dl>
   );
 }
+
+const projects = [
+  {
+    name: "ZaneOps",
+    updated_at: "Jan 13, 2024",
+    status: "0/5 Services Up",
+    actions: "Settings",
+    statusIcon: <X size={14} />,
+    tracker: 0
+  },
+  {
+    name: "ZaneOps",
+    updated_at: "Jan 13, 2024",
+    status: "5/5 Services Up",
+    actions: "Settings",
+    statusIcon: <Check size={14} />,
+    tracker: 1
+  },
+  {
+    name: "ZaneOps",
+    updated_at: "Jan 13, 2024",
+    status: "2/5 Services Up",
+    actions: "Settings",
+    statusIcon: <AlertTriangle size={14} />,
+    tracker: 2
+  }
+];
+
+export function ProjectList() {
+  return (
+    <main>
+      <div className="md:my-10 my-5">
+        <h1 className="text-3xl  font-bold">Overview</h1>
+        <h4 className="text-sm mt-2 opacity-60">List of projects</h4>
+      </div>
+
+      <div className="flex my-3 flex-wrap items-center md:gap-3 gap-1">
+        <div className="flex md:my-5 md:w-[30%] w-[100%]  items-center">
+          <Search size={20} className="relative left-5" />
+          <Input
+            className="px-14 placeholder:text-gray-400  -mx-5 w-full my-1 text-sm focus-visible:right-0"
+            placeholder="Ex: ZaneOps"
+          />
+        </div>
+
+        <div className="md:w-fit w-full">
+          <Menubar className="border border-border md:w-fit w-full ">
+            <MenubarMenu>
+              <MenubarTrigger className="flex md:w-fit w-full ring-secondary md:justify-center justify-between text-sm items-center gap-1">
+                Status
+                <ChevronsUpDown className="w-4" />
+              </MenubarTrigger>
+              <MenubarContent className=" border min-w-[23rem] border-border md:min-w-6">
+                <MenubarContentItem icon={Rocket} text="Active" />
+                <MenubarContentItem icon={Trash} text="Archived" />
+              </MenubarContent>
+            </MenubarMenu>
+          </Menubar>
+        </div>
+      </div>
+
+      <Table>
+        <TableHeader className="bg-toggle">
+          <TableRow className="border-none">
+            <TableHead>Name</TableHead>
+            <TableHead className="flex items-center gap-2">
+              Last Updated
+              <ArrowDown size={15} />
+            </TableHead>
+            <TableHead>Status</TableHead>
+            <TableHead className="text-right">Actions</TableHead>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          {projects.map((project) => (
+            <TableRow
+              className="border-border cursor-pointer"
+              key={project.name}
```

Comment:
There will be a project ID returned by the API FYI, so you should use that instead of the project's name for the key, as there can be two projects with the same name.

Reply:
okay for sure